### ISSUE TITLE
QUEUE Table/Detail View/queue subtable

### ISSUE THREAD

**Issue by leobardoramon** [gosha1128/NYUAMS#604]
<img width="1435" alt="screen shot 2016-08-24 at 2 32 15 am" src="https://cloud.githubusercontent.com/assets/2165796/17921285/22e621c8-69a7-11e6-8dc9-0f37689f2d78.png">

When user selects the either (a) parts or (b) order, the user will be able to either move the (a) a single part or (b) all of the part in an entire order to one of four different positions in the queue ( up , down , top , bottom )

https://projects.invisionapp.com/d/main#/console/8382217/179728678/preview

**Comment by gosha1128** [comment]
is it approved ?

**Comment by leobardoramon** [comment]
-APPROVED-

**Comment by gosha1128** [comment]
If you don't mind - I'd like split this task up into two.  First, to make sure the functionality of moving up/down works - I will temporarily add move up/down buttons to the table i have now.  We make sure it works correctly...then I will code it as the sub-tables you have designed.  Sound good?

**Comment by leobardoramon** [comment]
Yes, that is fine.

FYI, the "move up" and "move down" features exist in the current Devices tab

**Comment by gosha1128** [comment]
Yes, it works for a part I believe, but not for an order's parts.

**Comment by gosha1128** [comment]
OK, as i indicated before - i would like to proceed with this issue in 3 steps 1) functionality for move part 2) functionality for move order parts 3) the UI for these things

So for #1 - can you please test move parts ?  I temporarily put a move up/down button in the main queue table.  Does it work as you would expect ?

**Comment by leobardoramon** [comment]
Yes, it DOES work as expected.  Please proceed with next step.

**Comment by gosha1128** [comment]
OK, for moving an 'order up/down' - how does that work ?  It tries to move all parts for the order 1 position up/down ?

**Comment by leobardoramon** [comment]
The idea is that it keep all of the parts grouped together.  Yes, it would move all parts for one order up/down.  The most beneficial part of this feature would be "move to top" or "move to bottom" buttons for moving an entire order.

**Comment by gosha1128** [comment]
Ok let me do 'move all order parts to top'.  In combo with 'move
part' Hopefully that suffices

**Comment by leobardoramon** [comment]
QA Testing is complete.

Please continue Development. TY

**Comment by gosha1128** [comment]
yeah...move up and move down are super slow for some reason...so i need to work on that because it will affect move order up and down...
